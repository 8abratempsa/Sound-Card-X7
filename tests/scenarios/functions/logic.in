octosql "SELECT not(false), not(true),
                true AND false, true AND true, false AND false, false AND true,
                true OR false, true OR true, false OR false, false OR true,
                true OR NULL, false OR NULL,
                true AND NULL, false AND NULL"
