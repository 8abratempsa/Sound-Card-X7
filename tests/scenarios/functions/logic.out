+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+--------+--------+--------+--------+
| col_0 | col_1 | col_2 | col_3 | col_4 | col_5 | col_6 | col_7 | col_8 | col_9 | col_10 | col_11 | col_12 | col_13 |
+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+--------+--------+--------+--------+
| true  | false | false | true  | false | false | true  | true  | false | true  | true   | <null> | <null> | false  |
+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+--------+--------+--------+--------+
